internal/venice-common/src/main/resources/avro/StoreMetaValueWriteOpRecord.avsc

```diff
@@ -150,6 +150,40 @@
               "doc": "Properties related to Hybrid Store behavior. If absent (null), then the store is not hybrid.",
               "default": null
             },
+            {
+              "name": "views",
+              "doc": "A map of views which describe and configure a downstream view of a venice store. Keys in this map are for convenience of managing configs.",
+              "type": {
+              "type":"map",
```

Comment:
Space here？ Besides, a bit confused about the managing configs by using map instead of set here, could you explain a bit.

Reply:
Right, so map is preferred because otherwise it's pretty difficult to interact with on the admin api.  Also, after talking more about views in general with Felix and other folks, it seems like it would be useful to be able to specify a view either in Davinci or the change capture client down the road.  So a handle is very useful in all these cases.

Reply:
I mean indentation needed here for `"type": "map",` line. Just wondering if we can explain more in doc with examples about keys.

Reply:
Added.